pkgs/development/python-modules/robotframework-selenium2library/default.nix

```diff
@@ -1,29 +1,22 @@
-{ stdenv
-, buildPythonPackage
-, fetchPypi
-, robotframework
-, selenium
-, docutils
-, decorator
-}:
+{ stdenv, buildPythonPackage, fetchPypi, robotframework-seleniumlibrary }:
 
 buildPythonPackage rec {
-  version = "1.6.0";
+  version = "3.0.0";
   pname = "robotframework-selenium2library";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1asdwrpb4s7q08bx641yrh3yicgba14n3hxmsqs58mqf86ignwly";
+    sha256 = "2a8e942b0788b16ded253039008b34d2b46199283461b294f0f41a579c70fda7";
   };
 
-  # error: invalid command 'test'
-  #doCheck = false;
+  # Neither the PyPI tarball nor the repository has tests
+  doCheck = false;
 
-  propagatedBuildInputs = [ robotframework selenium docutils decorator ];
+  propagatedBuildInputs = [ robotframework-seleniumlibrary ];
 
   meta = with stdenv.lib; {
     description = "Web testing library for Robot Framework";
```

Comment:
Should the description indicate, that this is for helping transitioning and that the new name is `robotframework-seleniumlibrary`?

Reply:
I'm not sure, I think adding a to the project homepage should be enough; but if you prefer it I can add it to `longDescription`